pkg/i2gw/providers/ingressnginx/resource_reader.go

```diff
@@ -42,7 +44,14 @@ func (r *resourceReader) readResourcesFromCluster(ctx context.Context) (*storage
 	if err != nil {
 		return nil, err
 	}
-	storage.Ingresses = ingresses
+	ingNames := []types.NamespacedName{}
+	for ing := range ingresses {
+		ingNames = append(ingNames, ing)
+	}
+	sort.Slice(ingNames, func(i, j int) bool {
+		return ingNames[i].Name < ingNames[j].Name
+	})
+	storage.Ingresses = OrderedIngressMap{ingressNames: ingNames, ingressObjects: ingresses}
```

Comment:
I think having this functionality as a method/constructor of the `OrderedIngressMap` type will make things easier to maintain, especially as we already duplicate it later in this file.

Reply:
Done, Thanks!